sal_interfaces/CCArchiver/CCArchiver_Commands.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://project.lsst.org/ts/sal_objects/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://project.lsst.org/ts/sal_objects/schema/SALCommandSet.xsd">
+<SALCommand>
+    <Subsystem>CCArchiver</Subsystem>
+    <Version>3.7.2</Version>
+    <Author></Author>
+    <EFDB_Topic>CCArchiver_command_resetFromFault</EFDB_Topic>
+    <Alias>resetFromFault</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>ignored</Description>
```

Comment:
The correct description for an ignored parameter is “Attribute required by the API, but is unused.”

Reply:
This was replicated from the ATArchiver.  I'll update them from "ignored" to "Attributed required by the API, but is unused."